### ISSUE TITLE
network naming on BitGet

### ISSUE THREAD

**Issue by seeyoul8r** [ccxt/ccxt#23989]
### Operating System

windows 11

### Programming Languages

Python

### CCXT Version

last

### Description

Hello!
Asset: OCT | Please check network naming on BitGet, it should be BTC instead BRC20

### Code

```
  

```

**Comment by sc0Vu** [comment]
@seeyoul8r BRC20 is the name of token spec on BTC. Do you have any issues when use these names?

**Comment by seeyoul8r** [comment]
> @seeyoul8r BRC20 is the name of token spec on BTC. Do you have any issues when use these names?

On the other exchanges this two tokens has BTC networks. I guess they should be the same?

**Comment by kroitor** [comment]
> > @seeyoul8r BRC20 is the name of token spec on BTC. Do you have any issues when use these names?
> 
> On the other exchanges this two tokens have BTC networks. I guess they should be the same?

You're right! @sc0Vu can you please check other exchanges to make sure it is unified across all of them?

**Comment by seeyoul8r** [comment]
Hello! I found new inconsisntency:
LSD :
Bitget  APTOS 
BingX APT

**Comment by seeyoul8r** [comment]
> @seeyoul8r we will update the bitget inconsistency, thanks for reporting it. Can you please specify which exchanges are returning BTC instead of BRC20?

Coin COM on Bybit has BTC network. I guess you should also update manual / Network section, it says that network should be BTC, not BRC20
![Screenshot 2024-10-16 165120](https://github.com/user-attachments/assets/013976b6-e5eb-40c8-ae82-d69b18db11f8)


I found another inconsistency on Bybit:
Coin: Elixir | Bybit | Network: SPL instead SOL

**Comment by seeyoul8r** [comment]
Another one:
GAME
Kucoin ARBONE 
MEXC ARB

**Comment by seeyoul8r** [comment]
Another one:
CATCH
Kucoin ARBEVM
MEXC ARB

**Comment by carlosmiei** [comment]
@seeyoul8r Thanks we will try to update them shortly.

**Comment by seeyoul8r** [comment]
I've found new one:

CANTO 
Bitget CANTO-EVM 
Gate  CANTO

**Comment by batu3H** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by andrewljashenko** [comment]
ICX
Binance ICX
Kucoin ICON

ANIME
Binance ARBITRUM
Kucoin ARBONE
XT ARB
okx ARBONE

Regarding to ccxt on python.
Network Structure (https://docs.ccxt.com/#/README?id=network-structure) for Kucoin exchange hasn't 'network' key.

`'networks': {'ERC20': {'info': {'chainName': 'ERC20', 'withdrawalMinSize': '516', 'depositMinSize': None, 'withdrawFeeRate': '0', 'withdrawalMinFee': '258', 'isWithdrawEnabled': True, 'isDepositEnabled': True, 'confirms': 96, 'preConfirms': 32, 'contractAddress': '0x4dc26fc5854e7648a064a4abd590bbe71724c277', 'withdrawPrecision': 8, 'maxWithdraw': None, 'maxDeposit': None, 'needTag': False, 'chainId': 'eth'}, 'id': 'eth', 'name': 'ERC20', 'code': 'ERC20', 'active': True, 'fee': 258.0, 'deposit': True, 'withdraw': True, 'precision': 1e-08, 'limits': {'withdraw': {'min': 516.0, 'max': None}, 'deposit': {'min': None, 'max': None}}}}`

Also whitebit.currency() response has no 'networks' key at all.

**Comment by kroitor** [comment]
Closing this out with everything resolved or tracked — and thank you both for the meticulous collecting, this thread drove real unification work 🙏 Status of every item raised:

**Resolved by today's network-code unification** (https://github.com/ccxt/ccxt/pull/29601): the entire Arbitrum family — `ARBITRUM` is now the unified code across all exchanges, so the GAME/ANIME/kucoin/mexc inconsistencies are gone at the root. **Fixed earlier:** bitget APTOS→APT. **BRC20:** stands as the correct unified code (a distinct token standard on BTC, not the BTC network). **Fixed in https://github.com/ccxt/ccxt/pull/29610:** gate `ZKSYNC` (raw chain `ZKSERA`, live-verified today), bybit `SPL→SOL`, bitget `CANTO` (raw chain `CANTO-EVM`, live-verified today). **Verified already-correct on current master:** kucoin ICX — live check shows `chainId: icx` resolving to unified `ICX`; @andrewljashenko if you still see `ICON` on the latest version, please post a fresh dump:

```python
import ccxt

print('ccxt v' + ccxt.__version__)

exchange = ccxt.kucoin()
exchange.load_markets()
# exchange.verbose = True  # uncomment for debugging

print(list(exchange.currencies['ICX']['networks'].keys()))  # expected: ['ICX']
```

**Tracked on the ongoing unification audit:** the kucoin network-structure key naming and whitebit's missing `networks` — structural items riding separate PRs. Closing as completed; new inconsistencies deserve a fresh issue 🙏
